cmd/kube-proxy/app/server.go

```diff
@@ -209,6 +209,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet) {
 
 	fs.Float32Var(&o.config.ClientConnection.QPS, "kube-api-qps", o.config.ClientConnection.QPS, "QPS to use while talking with kubernetes apiserver")
 	fs.Var(&o.config.DetectLocalMode, "detect-local-mode", "Mode to use to detect local traffic. This parameter is ignored if a config file is specified by --config.")
+	fs.StringVar(&o.config.DetectLocal.BridgeInterfaceName, "pod-bridge-interface-name", o.config.DetectLocal.BridgeInterfaceName, "A bridge interface name in the cluster. Kube-proxy considers traffic as local if originating from an interface which matches the value. This argument should be set if DetectLocalMode is set to LocalModeBirdgeInterfaceName.")
```

Comment:
s/LocalModeBirdgeInterfaceName/LocalModeBridgeInterfaceName/

Reply:
actually, this is not the name of the value, is the name of the constant used internally.
We should use the name the users should use 

```
	LocalModeClusterCIDR         LocalMode = "ClusterCIDR"
	LocalModeNodeCIDR            LocalMode = "NodeCIDR"
	LocalModeBridgeInterfaceName LocalMode = "BridgeInterfaceName"
	LocalModeInterfacePrefix     LocalMode = "InterfacePrefix"
```

Reply:
Ah I made a mistake in the doc string . Will fix that (on PTO rn) next week. I’m not sure I follow the second comment? Since we are sub-configuring it within DetectLocal it doesn’t make sense to add Local again right ? cc @danwinship

Reply:
Sorry ignore my previous comment. I get what you meant. I’m guessing the last word in the doc string needs to be changed to BridgeInterfaceName since that’s what users will set.

Reply:
right

Reply:
done.